26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -64,3 +65,24 @@ Feature: Creator navigates to Description tab
     Given Creator begins Wizard registration and navigates to Description tab
     When their Institution has a Vocabulary set as "Default"
     Then they can see an input field for the Default Vocabulary
+
+  Scenario: Creator opens Dialog for creating a new Project
+    Given Creator begins Wizard registration and navigates to Description tab
+    When they click Button for creating a new Project
+    Then they see a Dialog with input fields:
+      | Project title            |
+      | Coordinating Institution |
+      | Project manager          |
+      | Start date               |
+      | Internal reference       |
+    And they see a Cancel Button
+    And they see a Save Button
+
+  @xxx
+  Scenario: Creator creates a new Project
+    Given Creator opens Dialog for creating a new Project
+    When they enter a Project title
```

Comment:
And they look up a institution as Coordinating Institution
And ​they look up a person as Project Manager
And they set a Start date